### PR TITLE
SIMULATION: follow trajectory perfectly in gazebo

### PR DESCRIPTION
**Pull request by kev-the-dev**
* disable buoyency/turbulence/thruster simulation so the position of SubjuGator is always the latest pose generated by c3

Addresses #231

### CODE REVIEW

File: simulation/sub8_gazebo/models/sub8/sub8.sdf [hunk 1]
```diff
@@ -172,7 +173,14 @@
         </plugin>
       </sensor>
     </link>
+    <!-- Commend out to enable physics simulated controls -->
```
Comment (DSsoto) [inline simulation/sub8_gazebo/models/sub8/sub8.sdf]: Comment

File: simulation/sub8_gazebo/models/sub8/sub8.sdf [hunk 2]
```diff
@@ -172,7 +173,14 @@
...
+    <plugin name="sub8_state_set" filename="libsub8_state_set.so">
+      <staticOffset>0 0 0</staticOffset>
+      <referenceTopic>/c3_trajectory_generator/trajectory_v</referenceTopic>
+      <alwaysOn>true</alwaysOn>
+    </plugin>
 
+    <!-- Uncomment to enable physics simulated controls
```
Comment (DSsoto) [inline simulation/sub8_gazebo/models/sub8/sub8.sdf:183]: Contradicts above comment

File: simulation/sub8_gazebo/launch/duck.launch [hunk 3]
```diff
@@ -4,28 +4,14 @@
...
+  <arg name="thrusters" default="false" />
 
 
   <!-- start gazebo server-->
   <node name="gazebo" pkg="gazebo_ros" type="gzserver" respawn="false"
     args="$(find sub8_gazebo)/worlds/a_whole_new.world" />
   <node name="gazebo_gui" pkg="gazebo_ros" type="gzclient" respawn="false" if="$(arg gui)" />
 
-
-  <!-- We need to spawn a torpedo that the launcher script can use to shoot -->
```
Comment (DSsoto) [inline simulation/sub8_gazebo/launch/duck.launch]: I don't know why you're removing this.

**Review by DSsoto — CHANGES_REQUESTED** [review]
Undo the removal of the torpedo board and I'll be ready to approve if it runs on my machine without issues. (will do tomorrow)

File: simulation/sub8_gazebo/models/sub8/sub8.sdf (hunk 2 above)
Comment (kev-the-dev) [reply simulation/sub8_gazebo/models/sub8/sub8.sdf:183]: If you want to simulate controls with physics, you comment out the state publisher and uncomment the buoyancy and thruster plugins.

Comment (DSsoto) [reply simulation/sub8_gazebo/models/sub8/sub8.sdf:183]: My mistake, I thought they were referring to the same code.

**Review by DSsoto — APPROVED** [review]

**Comment by DSsoto** [comment]
Ran sim with changes, looks good!

**Pull request merged**